src/cli/cli_dataset.cpp

```diff
@@ -527,6 +527,13 @@ ThreadError Dataset::ProcessMgmtSetCommand(otInstance *aInstance, int argc, char
             SuccessOrExit(error = Interpreter::ParseLong(argv[++index], value));
             dataset.mChannel = static_cast<uint16_t>(value);
         }
+        else if (strcmp(argv[index], "channelmask") == 0)
```

Comment:
Should we also need to append channel mask tlv in `SendSetRequest()` if this tlv is set by CLI?